projects/storefrontstyles/scss/themes/calydon/components/checkout/order-confirmation/_order-confirmation-thank-you-message.scss

```diff
@@ -1,7 +1,5 @@
 %cx-order-confirmation-thank-you-message-calydon {
-
   #cx-thank-you-header {
```

Comment:
@LTiger14 @reyhanehsalkhi 
I came across this PR while reviewing the Praetorian's board, sorry to cut in, but just wanted to comment on the fact that we're not using css ID selectors anywhere in the Design System, so i would suggest changing this selector to a class selector like `.cx-thank-you-header` ... the sass placeholder makes it specific enough already anyway.